### PR TITLE
refactor(console): add skeleton for custom domain page

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

- Add form card skeleton
- Extract the form card layout from the original form card and shared it with its skeleton
- Refactor the details page skeleton with the form card skeleton
- Add skeleton for custom domain page

## Testing

Test locally.

image

## Checklist

- [ ] `.changeset`
- [ ] unit tests
- [ ] integration tests
- [ ] docs

OR

- [x] This PR is not applicable for the checklist

### CODE REVIEW

File: packages/console/src/components/FormCard/FormCard/index.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import * as styles from './index.module.scss';
+
+type Props = {
+  title: AdminConsoleKey;
+  description?: AdminConsoleKey;
+  learnMoreLink?: string;
+  children: ReactNode;
+};
+
+function FormCard({ title, description, learnMoreLink, children }: Props) {
```
Comment (charIeszhao) [inline packages/console/src/components/FormCard/FormCard/index.tsx:19]: The component name is redundant. Consider renaming it to `FormCardLayout`

Comment (charIeszhao) [reply packages/console/src/components/FormCard/FormCard/index.tsx:19]: Ah, why do you extract this component again? Don't quite get it

File: packages/console/src/components/FormCard/components/FormCardLayout/index.tsx
Comment (charIeszhao) [inline packages/console/src/components/FormCard/components/FormCardLayout/index.tsx:1]: As we discussed last time, the parent folder `components` can be removed, I think

File: packages/console/src/components/FormCard/Skeleton/index.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import FormCardLayout from '../components/FormCardLayout';
+
+import * as styles from './index.module.scss';
+
+type Props = {
+  formFiledCount?: number;
```
Comment (charIeszhao) [inline packages/console/src/components/FormCard/Skeleton/index.tsx:6]: Typo: filed => field

File: packages/console/src/components/FormCard/components/FormCardLayout/index.tsx
Comment (xiaoyijun) [reply packages/console/src/components/FormCard/components/FormCardLayout/index.tsx:1]: The `FormCardLayout` is shared between the `Skeleton` and `FormCard`

File: packages/console/src/components/FormCard/FormCard/index.tsx (hunk 1 above)
Comment (xiaoyijun) [reply packages/console/src/components/FormCard/FormCard/index.tsx]: updated.

File: packages/console/src/components/FormCard/components/FormCardLayout/index.tsx
Comment (xiaoyijun) [reply packages/console/src/components/FormCard/components/FormCardLayout/index.tsx:1]: updated.

**Review by charIeszhao — APPROVED** [review]

**Pull request merged**